src/Tools/dotnet-monitor/HostBuilderHelper.cs

```diff
@@ -112,6 +138,22 @@ public static IHostBuilder CreateHostBuilder(HostBuilderSettings settings)
                 });
         }
 
+        private static void UpdatePropertiesWarningList(HostBuilderContext context, string warningMessage)
+        {
+            if (context.Properties.TryGetValue(Strings.Key_WarningMessages, out object errorList))
```

Comment:
Do not make the key a localizable property; it should be a string constant.

Reply:
Won't that lead us to having two distinct string constants (one in `HostBuildHelper`, one in `Startup`) that need to be kept in sync?

Reply:
I would suggest something like:
``` c#
class HostBuilderHelper
{
   public const string ResultKey = "DotnetMonitorHostBuilderResults";
}

Then you can use HostBuilderHelper.ResultKey in both classes